modules/nf-core/ensemblvep/download/ensemblvep-download.diff

```diff
@@ -0,0 +1,59 @@
+Changes in module 'nf-core/ensemblvep/download'
+--- /dev/null
++++ modules/nf-core/ensemblvep/download/nextflow.config
+@@ -0,0 +1,20 @@
++/*
++~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
++    annotation-cache/downloadvepcache ENSEMBLVEP_DOWNLOAD base config file
++~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
++    A 'blank slate' config file, appropriate for general use on most high performance
++    compute environments. Assumes that all software is installed and available on
++    the PATH. Runs in `local` mode - all jobs will be run on the logged in environment.
++----------------------------------------------------------------------------------------
++*/
++
++process {
++    withName: 'ENSEMBLVEP_DOWNLOAD' {
```

Comment:
Wait isn't this the same as the associated `nextflow.confg` alongisde the module files?

Reply:
yeah, that diff was created by the nf-core modules patch command, so no idea what happened there